app/models/manageiq/providers/vmware/infra_manager/inventory/collector/property_collector.rb

```diff
@@ -1,4 +1,30 @@
 module ManageIQ::Providers::Vmware::InfraManager::Inventory::Collector::PropertyCollector
+  def create_property_filter
+    property_filter
+  end
+
+  def property_filter
```

Comment:
mixed on caching this.

the connection - I'm kinda against it.
But here, this may make more sense to cache it